src/openrct2/object/SceneryGroupObject.h

```diff
@@ -9,6 +9,7 @@
 
 #pragma once
 
+#include "../peep/Staff.h"
```

Comment:
Move this include to the source file and add this after line 18:

```c++
enum class EntertainerCostume : uint8_t;
```

This is a forward declaration, so it compiles properly without us having to pull in all dependencies of `Staff.h`

Reply:
Done!